config/initializers/inflections.rb

```diff
@@ -4,20 +4,13 @@
 # are locale specific, and you may define rules for as many different
 # locales as you wish. All of these examples are active by default:
 # ActiveSupport::Inflector.inflections(:en) do |inflect|
-#   inflect.plural /^(ox)$/i, '\1en'
-#   inflect.singular /^(ox)en/i, '\1'
-#   inflect.irregular 'person', 'people'
+#   inflect.plural /^(ox)$/i, "\\1en"
+#   inflect.singular /^(ox)en/i, "\\1"
+#   inflect.irregular "person", "people"
 #   inflect.uncountable %w( fish sheep )
 # end
 
 # These inflection rules are supported but not enabled by default:
 # ActiveSupport::Inflector.inflections(:en) do |inflect|
-#   inflect.acronym 'RESTful'
+#   inflect.acronym "RESTful"
 # end
-
-ActiveSupport::Inflector.inflections(:en) do |inflect|
-  inflect.acronym 'OpenID'
-  inflect.acronym 'SAML'
-  inflect.acronym 'PKCE'
-  inflect.acronym 'JWT'
-end
```

Comment:
these inflections were only used to titleize something in a fancy way, which is a bit too clever in my opinion, and ran into some issues with autoloading from the SAML Idp gem. so i axed them in favor of a more explicit helper method to titleize correctly in the edit form.